python.pri

```diff
@@ -29,58 +29,51 @@
 # variable to specify an absolute path to a Python binary to use.
 # Setting this environment variable overrides any querying.
 
-win32 {
-	PYTHON = $$(MUMBLE_PYTHON)
-	isEmpty(PYTHON) {
+PYTHON = $$(MUMBLE_PYTHON)
+# If the MUMBLE_PYTHON environment variable wasn't set,
+# determine if the 'which' tool is available. We need it
+# to determine which python binary on the system to use.
+# If 'which' isn't available, error out.
+isEmpty(PYTHON) {
+	# Calling 'which which' is intentional.
+	# If which is available, it will print the path to it.
+	# If it isn't available, it will not print anything.
+	WHICH=$$system(which which 2>/dev/null)
+	isEmpty(WHICH) {
+		message("The 'which' command is not available on the system. Unable to search for Python installations. Assuming 'python'... If this is not correct, please point the MUMBLE_PYTHON environment variable at a working Python 2 or Python 3 binary.")
```

Comment:
This message will always show on Windows, since it doesn't have 'which'.
We can't have that.